src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1522,6 +1533,11 @@ void MainWindow::slot_render_widget_context_menu(const QPoint& point)
     menu->exec(point);
 }
 
+void MainWindow::emit_signal_crop_window_cleared() const {
```

Comment:
I believe this method should be renamed to something like `emit_refresh_attribute_editor_signal()`.